docs/use-mainnet/bridges-of-linea/how-to-bridge-erc20-tokens.mdx

```diff
@@ -42,6 +43,14 @@ We recommend that only tech operators that are providing liquidity use our bridg
 
 6. You now can bridge this token, and it will be saved in your token list for future bridging.
 
+:::note
```

Comment:
Hey, thank you for adding this in the doc!
The chance this happens to a random end-users is very small, as this would only occur with exotic ERC20. So I think we should stress this case is very unlikely not to scare or confuse users.

Reply:
sounds good. added that into the docs in both places.